docs/release-notes/NuGet-6.0.md

```diff
@@ -0,0 +1,280 @@
+
+* Package Namespaces: Account for package namespaces filtering in package downloading in PackageReference restore - [#10732](https://github.com/NuGet/Home/issues/10732)
+
+* Add namespaces filtering support in packages.config restore in nuget.exe and msbuild /t:restore - [#10737](https://github.com/NuGet/Home/issues/10737)
+
+* Add a contract for identifying a restore source, to help nuget better batch restores/branch switches.  - [#10807](https://github.com/NuGet/Home/issues/10807)
+
+* Add support for net6.0-tizen and net6.0-android - [#10819](https://github.com/NuGet/Home/issues/10819)
+
+* Add namespaces filtering support for packages.config restore in Visual Studio - [#10823](https://github.com/NuGet/Home/issues/10823)
+
+* Add support for TfmSpecificDebugSymbolsFile to provide symbols in the inner build  - [#10913](https://github.com/NuGet/Home/issues/10913)
+
+* Show warning icon in Installed tab if there are vulnerable packages installed - [#10982](https://github.com/NuGet/Home/issues/10982)
+
+* Show warning icon by package in list of packages if there is a vulnerability - [#10983](https://github.com/NuGet/Home/issues/10983)
+
+* Show package vulnerability details in the package details pane of the PMUI - [#10985](https://github.com/NuGet/Home/issues/10985)
+
+* Add Deprecation Links in Details page in PM UI - [#10996](https://github.com/NuGet/Home/issues/10996)
+
+* [Feature]: Make retry count and backoff behavior configurable for Nuget Clients  - [#11027](https://github.com/NuGet/Home/issues/11027)
+
+* [Feature]: Address all the package installation rules for package namespaces work in PackageReference and PackageDownload scenarios. - [#11035](https://github.com/NuGet/Home/issues/11035)
+
+* [Feature]: Address all the package installation rules for package namespaces work in Packages Config style projects - [#11036](https://github.com/NuGet/Home/issues/11036)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* Consider removing the extensibility that allows other package managers to be shown as compatible - [#6623](https://github.com/NuGet/Home/issues/6623)
+
+* Stop using EmbedInteropTypes for NuGet's VS extenbility packages/assemblies - [#10892](https://github.com/NuGet/Home/issues/10892)
+
+* Project.nuget.g.targets should not prepend to MSBuildAllProjects in MSBuild 16 or later - [#10895](https://github.com/NuGet/Home/issues/10895)
+
+* Remove obsolete core APIs - [#10940](https://github.com/NuGet/Home/issues/10940)
+
+* Merge NuGet.SolutionRestoreManager.Interop into NuGet.VisualStudio - [#10957](https://github.com/NuGet/Home/issues/10957)
+
+* Remove RuntimeEnvironmentHelper.IsDev14 - [#11000](https://github.com/NuGet/Home/issues/11000)
+
+* Update Newtonsoft.Json to 13.0.1 - [#11095](https://github.com/NuGet/Home/issues/11095)
+
+* [DCR]: Rename packageNamespaces feature to packageSourceMapping - [#11205](https://github.com/NuGet/Home/issues/11205)
```

Comment:
Remove this, it was a change within the original release, so as far as customers are concerned, they never knew about package namespaces to begin with.